config/locales/cy.yml

```diff
@@ -260,9 +260,12 @@ cy:
     hidden_accessibility: Ffurflen adborth yn agor mewn ffenestr newydd
 
   footer:
+    sfgb_part_of: Rhan o'r 
+    sfgb_singe_financial: Corff Sengl ar
```

Comment:
Typo in the yaml stopping the second line from coming in on the Welsh.
Also, not sure if caused by this, but it also says sfgb at the start in the Welsh
<img width="700" alt="screenshot 2018-12-18 at 10 31 32" src="https://user-images.githubusercontent.com/3898629/50148478-8592c980-02b0-11e9-8dea-5d38f82732cd.png">